src/deadline/nuke_submitter/assets.py

```diff
@@ -81,17 +81,41 @@ def get_scene_asset_references() -> AssetReferences:
             for filename in get_node_filenames(node):
                 asset_references.output_directories.add(dirname(filename))
 
-    # if using a custom OCIO config, add the config file and associated search directories
-    if nuke_ocio.is_custom_config_enabled():
-        ocio_config_path = nuke_ocio.get_custom_config_path()
-        ocio_config_search_paths = nuke_ocio.get_config_absolute_search_paths(ocio_config_path)
+    if nuke_ocio.is_OCIO_enabled():
+        # Determine and add the config file and associated search directories
+        ocio_config_path = get_ocio_config_path()
+        # Add the references
+        if ocio_config_path is not None:
+            if os.path.isfile(ocio_config_path):
+                asset_references.input_filenames.add(ocio_config_path)
+
+                ocio_config_search_paths = nuke_ocio.get_config_absolute_search_paths(
+                    ocio_config_path
+                )
+                for search_path in ocio_config_search_paths:
+                    asset_references.input_directories.add(search_path)
+            else:
+                nuke.alert(
+                    "OCIO config file specified(%s) is not an existing file." % ocio_config_path
+                )
 
-        asset_references.input_filenames.add(ocio_config_path)
+    return asset_references
 
-        for search_path in ocio_config_search_paths:
-            asset_references.input_directories.add(search_path)
 
-    return asset_references
+def get_ocio_config_path() -> Optional[str]:
+    ocio_config_path = None
+
+    # if using a custom OCIO environment variable
+    if nuke_ocio.is_env_config_enabled():
+        ocio_config_path = nuke_ocio.get_env_config_path()
+    else:
+        print("here?")
```

Comment:
Nit: remove

Reply:
Dang! I thought I erased all of it. haha Thanks for catching it.